src/steps/tags.ts

```diff
@@ -26,10 +26,12 @@ export function extractTagsFromPath(filePath: string) {
   // for filename take first part before dot to omit extension and sub-extension.
   const fileNamePart = path.basename(filePath).split('.')[0] || '';
   const dirParts = path.dirname(filePath).split(path.sep);
-  [...dirParts, fileNamePart].forEach((part) => {
-    // consider any @-prefixed symbols as tag
-    const partTags = part.match(/@[^@\s]+/g) || [];
-    tags.push(...partTags);
-  });
+  if (!filePath.includes(`node_modules${path.sep}`)) {
```

Comment:
1. Suggestion: move this check upper for earlier return without computing fileNamePart and dirParts.
2. Question: why not `${path.sep}node_modules${path.sep}`?

Reply:
Do you prefer if I create this function in `utils/paths.ts` or directly in `tags.ts`?